Client/Coordinators/CoordinatorFlagManager.swift

```diff
@@ -5,11 +5,23 @@
 import Foundation
 import Shared
 
-/// This is a temporary struct made to manage the feature flag for conveniance
-/// This will be removed with FXIOS-6036
+/// This is a temporary struct made to manage the coordinator multiple feature flag for conveniance
 struct CoordinatorFlagManager {
+    /// This will be removed with FXIOS-6036
     static var isCoordinatorEnabled: Bool {
         return FeatureFlagsManager.shared.isFeatureEnabled(.coordinatorsRefactor,
                                                            checking: .buildOnly)
     }
+
+    /// This will be removed with FXIOS-6529
+    static var isSettingsCoordinatorEnabled: Bool {
+        return FeatureFlagsManager.shared.isFeatureEnabled(.settingsCoordinatorRefactor,
+                                                           checking: .buildOnly)
+    }
+
+    /// This will be removed with FXIOS-6530
+    static var isLibraryCoordinatorEnabled: Bool {
+        return FeatureFlagsManager.shared.isFeatureEnabled(.libraryCoordinatorRefactor,
+                                                           checking: .buildOnly)
+    }
```

Comment:
Since these are dependant on the above flag you could add a layer of safety by first checking isCoordinatorEnabled.